src/Accounts/Authentication/Authentication.csproj

```diff
@@ -12,10 +12,11 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Azure.Identity" Version="1.6.1" />
+    <PackageReference Update="Azure.Core" Version="1.36.0" />
```

Comment:
`Azure.Core` is referenced in our common targets file `tools\Common.Netcore.Dependencies.targets`

Reply:
Yeah, these shouldn't be here. Thanks for catching those.